### ISSUE TITLE
Maintainers needed

### ISSUE THREAD

**Issue by dandv** [SortableJS/sortablejs-meteor#2]
I'm the original author of the Meteor integration integration for rubaxa:sortable. 

Unfortunately I don't have much time nowadays to dedicate to this project. The code though is very well documented, so I hope that maintainers will be found.

**Comment by thoni56** [comment]
Agreed. It seems like a very valuable meteor packages, which unfortunately I could not get to work with the current (1.8) Meteor.

**Comment by jankapunkt** [comment]
The question is, if it makes sense to **a)** keep it heavily bound to Blaze or to **b)** make it Meteor compatible but frontend agnostic. The second option could at least attract more people from the Meteor community.
